docs/platforms/react-native/features/index.mdx

```diff
@@ -1,11 +1,15 @@
+---
+title: Features
+sidebar_order: 1
+description: "Learn more about Sentry's React Native SDK features."
+---
+
 <Note>
 
 Sentry's React Native SDK enables automatic reporting of errors and exceptions, and identifies performance issues in your application. The React Native SDK includes performance monitoring (our `@sentry/tracing` package) by default.
 
 </Note>
```

Comment:
The `@sentry/tracing` package is deprecated over 50 JS feature versions and removed in v8. I think we can remove the note.